docker-local/postman/scripts/_01_seed_hub_account.sh

```diff
@@ -5,7 +5,7 @@ PATH=$(cd ${DIR} && npm bin):$PATH
 POSTMAN_DIR="${DIR}/.."
 
 newman run \
-  --delay-request=2000 \
+  --delay-request=100 \
```

Comment:
Let's make this an an environment variable, that defaults to 100

`NEWMAN_REQUEST_DELAY=${NEWMAN_REQUEST_DELAY:100}`

Reply:
done.